src/gapp_login/wechat/endpoints.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from urllib import parse
+
+from authlib_gino.fastapi_session.api import auth, login_context
+from authlib_gino.fastapi_session.models import User
+from fastapi import APIRouter, HTTPException, Form, FastAPI, Depends
+from starlette import status
+from starlette.requests import Request
+
+from . import config
+from .models import db, WeChatIdentity
+from .utils import WeChatAuthError
+
+log = logging.getLogger(__name__)
+router = APIRouter()
+
+
+@router.put("/login/wechat")
```

Comment:
If I use POST method here, I will get `AttributeError: 'FormData' object has no attribute 'update'` raised from https://github.com/python-gino/authlib-gino/blob/master/src/authlib_gino/starlette_oauth2/authorization_server.py#L60

Reply:
Ah it should be converted into a normal `dict`